### PR TITLE
Added mobile view for game

### PR DESCRIPTION
**Pull request by yousuf031991**
Changed style.css to add classes for mobile view.
Removed redundant flipclock.css

### CODE REVIEW

**Review by adityab17190 — APPROVED** [review]
Works well. +1!

File: MainApp/public/app/controllers/gamePageCtrl.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, $location, $anchorScroll, $rootScope,
+                            $cookies, PatientService, Room, Agent, Circle, GameState, UserStats, Refresh, Scrolling) {
         let app = this;
         let blinkTimer;
         let blinkTimer2;
         Refresh.checkRefresh();
         app.username=$rootScope.username;
         (function startButton() {
-            alert("The goal is to save as many patients as possible");
+            alert("The goal is to save as many patients as possible. Please switch to landscape mode if using a " +
```
Comment (super9user) [inline MainApp/public/app/controllers/gamePageCtrl.js:10]: Minor comment: Can we detect if current device is mobile and only then display "please switch to landscape
"

Comment (yousuf031991) [reply MainApp/public/app/controllers/gamePageCtrl.js:10]: I can detect screen width and do that. However, even for tablets the experience is much better on landscape. And some tablets like the IPad pro are almost as big as some laptop screens. Plus some people orient their monitors vertically - where again the experience is not as good as in landscape. So not exactly sure what screen width I should use.

Comment (super9user) [reply MainApp/public/app/controllers/gamePageCtrl.js:10]: Actually no need to check on width.
http://stackoverflow.com/questions/3514784/what-is-the-best-way-to-detect-a-mobile-device-in-jquery
Check the accepted answer.
P.S: this is a minor comment. So can be done after the demo as well, if deemed necessary.

Comment (yousuf031991) [reply MainApp/public/app/controllers/gamePageCtrl.js:10]: Let's do it later then. Merging unless there's anything else.

**Review by super9user — APPROVED** [review]
+1

**Pull request merged**